docs/cms-query-integration.md

```diff
@@ -0,0 +1,68 @@
+# CMS Query Integration
+
+The Search module overrides the default search functionality to query the specialized search index in place of a standard MySQL query. This means all default search operations using the CMS search APIs for posts, users and terms such as `WP_Query`, `WP_User_Query`, `WP_Term_Query` and the REST API search endpoints will transparently make use of the search index by default.
+
+It is possible to control this behaviour and thus use the search index for non search queries that might otherwise be too slow to do in the database such as meta queries.
+
+Each query class accepts an array of parameters. ElasticPress checks for the value of an `ep_integrate` parameter, if this is true the query is performed by Elasticsearch instead of MySQL. If the `s` (search) parameter is present `ep_integrate` will be set to `true` automatically.
+
+The following example sets `ep_integrate` to true for a non search query, that is any query where the `s` parameter is not used.
```

Comment:
Hyphenate "non search", add period, add comma after "that is"

```suggestion
The following example sets `ep_integrate` to true for a non-search query. That is, any query where the `s` parameter is not used.
```

I'm not sure i like the sentence starting with "That is", so perhaps this sentence can be shortened to:

```
The following example sets `ep_integrate` to true for a non-search query where the `s` parameter is not used.
```

or

```
The following example sets `ep_integrate` to true for a query where the `s` parameter is not used.
```

Since this is a "for example", it should probably end with a `:` instead of a period.

Reply:
I like the first replacement sentence example, good shout. I think it's important to call out "non-search" still

Reply:
```suggestion
The following example sets `ep_integrate` to true for a non-search query where the `s` parameter is not used.
```